lib/utilities/valid-project-name.js

```diff
@@ -1,11 +1,39 @@
 'use strict';
 
+/**
+ * Checks if the string starts with a number.
+ *
+ * @method startsWithNumber
+ * @return {Boolean}
+*/
+function startsWithNumber(name) {
+  return (/^\d.*$/).test(name);
```

Comment:
I just noticed that the regex could actually be simplified to `/^\d/` since we don't care about the stuff that's after the first character